applications/models.py

```diff
@@ -281,6 +281,17 @@ class WinterStorageApplication(BaseApplication):
         verbose_name=_("trailer registration number"), max_length=64, blank=True
     )
 
+    def is_unmarked_ws_application(self) -> bool:
```

Comment:
slightly more powerful result if you mark it as `@property`. you could use it as field in gql without custom resolver then

Reply:
I think we want to publish the field only when we create the new field for WS application type (the enum). So there is no reason to publish it now in gql or what do you think?

Reply:
no, setting it as property will nto automatically publish it, but it would make using it in someinstances easier. OTOH, we will probably add a new enum field, which will be published in the gql api, and not this property method, since it is more efficient that way, so not really that much use for the property except for checking the type before saving the new future enum field.